docs/index.rst

```diff
@@ -133,8 +133,8 @@ The API reference details the functionality of each ``class`` and ``function`` p
    :target: https://pypi.org/project/vector/
 .. |Conda latest releasetatus| image:: https://img.shields.io/conda/vn/conda-forge/vector.svg
    :target: https://github.com/conda-forge/vector-feedstock
-.. |DOI| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.5942083.svg
-   :target: https://doi.org/10.5281/zenodo.5942082
+.. |DOI| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.7054478.svg
```

Comment:
The badges on the website should be split into 2 lines, just as they are in the README!

Reply:
Did the splitting.